modules/ref/pages/client-settings.adoc

```diff
@@ -140,7 +146,7 @@ As an alternative to specifying the certificates to trust, you can specify a cus
 See the xref:howtos:managing-connections.adoc#ssl[Connection Management] section for more details on how to set it up properly.
 
 
-== I/O Options
+=== I/O Options
```

Comment:
You're losing most of the options from the rh-nav?

Reply:
Ah! Nope, hadn't noticed that...

But some options were h2, some h3, and didn't seem to be a hierarchy, so made sense to normalise them.

Could remove the "== Configuration Options" section entirely, and promote the children to h2?

Reply:
Okay, let's give it a go.